app/controllers/clearance/sessions_controller.rb

```diff
@@ -21,11 +22,17 @@ def destroy
   end
 
   def new
-    render template: 'sessions/new'
+    render template: "sessions/new"
   end
 
   private
 
+  def redirect_signed_in_users
+    if current_user
```

Comment:
`signed_in?` is probably better here. I think this will break if you have a guest user, for instance.